packages/nx-plugin/src/generators/configuration/code-pushup-config.ts

```diff
@@ -0,0 +1,71 @@
+import { Tree, generateFiles, logger } from '@nx/devkit';
+import { join } from 'node:path';
+import type { PersistConfig, UploadConfig } from '@code-pushup/models';
+import type { ItemOrArray } from '@code-pushup/utils';
+import { ExecutableCode } from './types';
+import {
+  formatArrayToJSArray,
+  formatArrayToLinesOfJsString,
+  formatObjectToFormattedJsString,
+  normalizeExecutableCode,
+  normalizeItemOrArray,
+} from './utils';
+
+export const DEFAULT_IMPORTS = [
+  "import type { CoreConfig } from '@code-pushup/models;'",
```

Comment:
Invalid syntax here:

```suggestion
  "import type { CoreConfig } from '@code-pushup/models';",
```